src/main/telemetry/ibus_shared.c

```diff
@@ -33,19 +33,41 @@
 #include "telemetry/telemetry.h"
 #include "telemetry/ibus_shared.h"
 
-static uint16_t calculateChecksum(const uint8_t *ibusPacket, size_t packetLength);
-
+static uint16_t calculateChecksum(const uint8_t *ibusPacket);
 
 #if defined(TELEMETRY) && defined(TELEMETRY_IBUS)
-
+#if defined(GPS) && FLASH_SIZE > 256
+#define TELEMETRY_IBUS_EXTENDED
```

Comment:
This should go into `target/common_fc_pre.h`, which is the centralised location for these target / flash size dependent defines.